consensus/docker-compose-files/scripts/infiniteloop.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Hi @adecaro,  we already have hyperledger/fabric/scripts/infiniteloop.sh  .

Are you replacing that copy of infiniteloop.sh ? or are you adding a new .sh script ?   
In any case, I think we're better off keeping hyperledger/fabric/scripts/infiniteloop.sh  and updating compose-consensus-4.yaml to put in the right path for the cli container ( `command: ../scripts/infiniteloop.sh` I think but verify that )